### PR TITLE
[REFACTOR] Styledown CSS and JS

### PR DESCRIPTION
**Pull request by KamiKillertO**
## JS changes

Styledown.js now longer support **ie**<9.
I've updated the the js to make the code reveal/hide button accessible. The button used to have no label, it's no longer the case and the label is updated on click to indique `Reveal code` or `Hide code`.

## CSS changes

The reveal/hide button contrast has been improved + now it has a nice blue border on focus.
Screenshot_20200414_111506

The default colors for highlightsjs has been updated to use almost the same colors has github.

Before
Screenshot_20200415_105908

Now
Screenshot_20200415_105644

## Default config

Because we've have forked Styledown we have to use our JS/CSS files by default instead of Styledown ones. The default conf used to include the Styledown JS/CSS using a cdn, now the content of those files are inlined in the default config. I've choose to inline them because it was the easiest solution and because they are both small it has not impact on the page loading time.

### CODE REVIEW

File: assets/styledown.css [hunk 1]
```diff
@@ -38,30 +42,27 @@ h3.sg {
...
+.sg-code,
+.sg-canvas,
+.sg-text {
   -webkit-box-sizing: border-box;
   -moz-box-sizing: border-box;
   box-sizing: border-box;
 }
 
-/**
+/* *
```
Comment (ryuran) [inline assets/styledown.css:65]: A space added, why ?

Comment (KamiKillertO) [reply assets/styledown.css:65]: Typo 😝

**Comment by KamiKillertO** [comment]
@ryuran I've added the before/after screenshots

**Comment by KamiKillertO** [comment]
@ainarela normally the impact on PDUI should be minimum as PDUI has some CSS to override the Styledown default colors. The only visible change will be the reveal/hide button

**Comment by ainarela** [comment]
@KamiKillertO I mean when we will se those changes in PDUI - as this is a different repo. Do we need to update any dependencies in PDUI?

**Comment by KamiKillertO** [comment]
@ainarela First we need to release a new version, then we need to :
1. Update broccoli-styledown to use the new version of Styledown
2. Release a new version of broccoli-styledown
3. Update PDUI 👍 

Also there's a new feature in Styledown that might help reducing the sizes of the SCSS files in PDUI (https://github.com/peopledoc/styledown/pull/19)

**Pull request merged**